frontend/index.html

```diff
@@ -15,3 +15,14 @@ <h1>This is an exapmle</h1>
 
   </body>
 </html>
+
+<script>
+    async function main () {
+        const res = await fetch('https://ctabustracker.com/bustime/api/v3/getvehicles?key=gbNab8LFcBWDx5qmtc8Q8dfjb&rt=172&format=json')
+        const json = await res.json()
+        console.log(json)
+    }
+ 
+ main()
+
+</script>
```

Comment:
seems this might have been left over from a previous experiment, can u discard this change?